doc/Core.xml

```diff
@@ -2562,14 +2568,27 @@ X-Frame-Options: SAMEORIGIN]]></programlisting>
         <section>
           <title>Default Access Policy</title>
           <para>By default, the device should enforce the following default access policy, which gives an acceptable level of security in many systems.</para>
-          <para>The default access policy builds upon the access classes that are associated to the services and grants access rights in the following way. A user of level c shall be granted access to a service associated to access class r if and only if an "X" is present in the cell at column c and row r in <xref linkend="_Ref292468385" />.</para>
+          <para>The default access policy builds upon the access classes that are associated to the services and grants access rights in the following way. A user of level c shall be granted access to a service associated to access class r if and only if an "X" is present in the cell at column c and row r in <xref linkend="_Ref292468385"/>.</para>
           <para>A device that signals support for the Default Access Policy via the respective capability shall support at least one user of each user level Administrator, Operator and User.</para>
         </section>
+        <section>
+          <title>Editable user levels</title>
```

Comment:
Should this section define the expected behavior if a device receives a function it does not know (ex. new API call added, but device running older firmware)

Reply:
If you do not mind, I'd prefer to add a dedicated error in §8.4.10.2 (i.e. Set editable user level)

Reply:
During the San Diego F2F meeting, we agreed that we are going to ignore the unknown function